### PR TITLE
Time Range Validation

### PR DESCRIPTION
**Pull request by relvacode**
Resolution for issue discussed in #6

### CODE REVIEW

**Review by Gaadek — APPROVED** [review]
Great update!

**Review by rickb777 — COMMENTED** [review]
Good work; I've made a few suggestions.

File: time.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	31 + 28 + 31 + 30,
+	31 + 28 + 31 + 30 + 31,
+	31 + 28 + 31 + 30 + 31 + 30,
+	31 + 28 + 31 + 30 + 31 + 30 + 31,
+	31 + 28 + 31 + 30 + 31 + 30 + 31 + 31,
+	31 + 28 + 31 + 30 + 31 + 30 + 31 + 31 + 30,
+	31 + 28 + 31 + 30 + 31 + 30 + 31 + 31 + 30 + 31,
+	31 + 28 + 31 + 30 + 31 + 30 + 31 + 31 + 30 + 31 + 30,
+	31 + 28 + 31 + 30 + 31 + 30 + 31 + 31 + 30 + 31 + 30 + 31,
+}
```
[the remark this answers is not in the document]
Comment (relvacode) [reply time.go]: Good call, updated

File: iso8601_test.go [hunk 2]
```diff
@@ -213,17 +252,60 @@ var cases = []TestCase{
...
+	},
+
+	{
+		Using:                   "2017-01-01T00:60:00.000+00:00",
+		ShouldInvalidRange:      true,
+		RangeElementWhenInvalid: "minute",
+	},
+
+	{
+		Using:                   "2017-01-01T00:00:60.000+00:00",
```
[the remark this answers is not in the document]
Comment (relvacode) [reply iso8601_test.go:302]: I've added a note in the README regarding leap seconds

**Review by rickb777 — APPROVED** [review]
This looks ready to merge. Thanks for your work.

**Pull request merged**